mantidimaging/gui/windows/main/test/presenter_test.py

```diff
@@ -565,7 +565,12 @@ def test_tabify_stack_window_to_item_in_list(self):
         self.view.tabifyDockWidget.assert_called_once_with(other_stack, new_stack)
 
     def test_create_strict_dataset_tree_view_items(self):
-        dataset = StrictDataset(*generate_images_with_filenames(5))
+        stacks = generate_images_with_filenames(5)
+        dataset = StrictDataset(sample=stacks[0],
```

Comment:
NITPICK, not that this really need condensing, but we could maybe get away with unpacking this instead:
`dataset = StrictDataset(sample=stacks[0], *stacks[1:5])`

Reply:
At least during the migration I want to force named arguments to avoid any ambiguity with the constructors.

Reply:
Definitely! I appreciate you adding * to do this to the initialisers